src/main/java/com/treasure/hunt/strategy/hider/Hider.java

```diff
@@ -24,5 +24,10 @@
     /**
      * @return the current treasure location
      */
-    Point getTreasureLocation();
+    Point getTreasurePos();
+
+    /**
+     * @param treasureDistance the distance between start position and treasure
+     * */
+    void setTreasureDistance(double treasureDistance);
```

Comment:
Since when do we need to make restrictions to the hider, which treasure distance he has to choose?

It causes extra efford, implementing this into each (and every new) Hider.

Reply:
There is no need to implement but hard coding the distance is very bad

Reply:
I don't understand the usage of this function. Does it give the hinter restrictions for placing the treasure location in a certain distance? @axel1200 please explain. Especially because we seem to need it in each and every hider. Maybe the javadoc needs a tweak or two more ;)

Reply:
Somehow we need to tell the Hider/ searcher

Reply:
Well we can remove it but we need to tell the hider somehow how big the playing field is instead of just throwing Exceptions like @DorianRec does

Reply:
yes, the searcher/hider need to know this!
1. I cogitate over giving them width and height via Constructor, since we does not change them.
2. But we can also put this into a init()/setter method for searcher/hider.

But throwing exceptions is the right way in this context in my mind, because we assume the hider/searcher following the rules. If they dont, we have to abort the game.

Reply:
Ok this will be fixed by #107